tests/ctst/features/azureArchive.feature

```diff
@@ -187,4 +187,34 @@ Feature: Azure Archive
         | versioningConfiguration | objectCount | objectSize | restoreDays |
         |           Non versioned |           2 |        100 |           5 |
         |               Versioned |           2 |        100 |           2 |
-        # |               Suspended |           2 |        100 |           2 | 
\ No newline at end of file
+        # |               Suspended |           2 |        100 |           2 | 
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    @ColdStorage
+    Scenario Outline: Cannot add object MD to a transitioned object
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-azure-archive"
+    Then i "should not" be able to add user metadata to object "obj-1"
+    When i restore object "obj-1" for 10 days
+    When i restore object "obj-2" for 10 days
+    Then blob for object "obj-1" must be rehydrated
+    Then blob for object "obj-2" must be rehydrated
+    Then object "obj-1" should be "restored" and have the storage class "e2e-azure-archive"
+    Then object "obj-2" should be "restored" and have the storage class "e2e-azure-archive"
+    Then i "should" be able to add user metadata to object "obj-1"
+    Then i "should" be able to add user metadata to object "obj-2"
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-azure-archive"
+    Then object "obj-2" should be "transitioned" and have the storage class "e2e-azure-archive"
```

Comment:
I'm afraid this test will pass even without having the same object, we should probably check before that it has the restoration fields (using a headObject).

And maybe we should adapt this step to make it say `... should be "expired ..."` to be clearer about what we expect.

Reply:
Since it's a new version we can leave it to transitioned. (I misunderstood what the copy object does and thought it was supposed to keep the same metadata and not create a new version)